libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1229,6 +1229,34 @@ export interface FileConsentCardResponse {
     uploadInfo?: FileUploadInfo;
 }
 
+/**
+ * @interface
+ * Tab response to 'task/submit'.
+ */
+export interface AdaptiveCardActionResponse {
```

Comment:
The [.NET implementation](https://github.com/microsoft/botbuilder-dotnet/blob/3c335046f95deeac50fbb0b48c7c8c42051d4f6d/libraries/Microsoft.Bot.Builder/ActivityHandler.cs#L514) reuses the [AdaptiveCardInvokeResponse](https://github.com/microsoft/botbuilder-dotnet/blob/3c335046f95deeac50fbb0b48c7c8c42051d4f6d/libraries/Microsoft.Bot.Schema/AdaptiveCardInvokeResponse.cs#L11) type, any reason we can't do the same here?

Reply:
Sorry I didn't see that type before. Thanks for the suggestion.